stats.go

```diff
@@ -30,10 +31,14 @@ func (s *Stats) toRedis() []interface{} {
 		strconv.FormatUint(atomic.LoadUint64(&s.producerUnflushed), 10),
 		"producer.delivery.errors",
 		strconv.FormatUint(atomic.LoadUint64(&s.producerErr), 10),
+		"producer.connection.errors",
```

Comment:
We could probably name this `producer.unknown.events` to be more clear on reports.